src/com/company/Player.java

```diff
@@ -0,0 +1,14 @@
+package com.company;
+
+public class Player {
+    String name;
```

Comment:
"private" access modifier

Reply:
Changed all instance variables to private and added getters/setters.